include/appbase/application.hpp

```diff
@@ -273,6 +282,10 @@ namespace appbase {
 
          std::unique_ptr<class application_impl> my;
 
+         bool should_reset { false };
+         
+         static std::unique_ptr<application> app_instance;
+         inline static std::vector<std::function<void ()>> plugin_registrations;
```

Comment:
Minor thing, but I would have expected this to have a `{}` at the end.
`inline static std::vector<std::function<void ()>> plugin_registrations{};`
It doesn't appear to be necessary, but seems cleaner to me.

Reply:
Why does it seem cleaner? You never need to specify an initializer for types that have default constructors.

Reply:
For `inline static` the initialization his here and so adding the `{}` makes that more obvious in my opinion.

Reply:
If you strongly prefer, I can add the `{}`, but because it really does not change anything, I think it is just extra noise. When I see things like that, I can't help wonder why it is there since it serves no purpose. A little like adding `return;` statements at the end of `void` returning functions. :-)

Reply:
Not going to push hard for this. My only thought was that `inline static` being new to C++-17 the explicit `{}` gives the user a bit more indication this is not your typical `static`. All docs I could find on use of `inline static` show an example with initialization, obviously that is not needed for `std::vector`, but `{}` provides it.

Reply:
FYI, from cppreference 
<img width="623" alt="image" src="https://user-images.githubusercontent.com/1496907/212409987-6c35602a-39de-4006-9f61-2bab7abe4a19.png">

Reply:
Yep, saw that.